cobalt/browser/memory_settings/auto_mem.cc

```diff
@@ -412,11 +413,29 @@ void AutoMem::ConstructSettings(const math::Size& ui_resolution,
   offscreen_target_cache_size_in_bytes_->set_memory_scaling_function(
       MakeLinearMemoryScaler(0.25, 1.0));
 
-  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
   std::string rasterizer_type =
       configuration::Configuration::GetInstance()->CobaltRasterizerType();
+  auto persistent_settings =
```

Comment:
Again, i recommend moving the code from here from lines 418 down to 447 , including commandline check, into a standalone function. Maybe even a single-file module like `resolve_rasterizer_settings.cc` 

It only needs to depend on a few things
 
- configuration::
- persistent_settings
- base::CommandLine
- browser::switches

To avoid the duplication here. I'm sure a reasonable place somewhere under `cobalt/browser exists for this.